### ISSUE TITLE
Parallel Recursion

### ISSUE THREAD

**Issue by slai7880** [uwhpsc-2016/lectures#30]
So far we have talked about building parallel codes based on the serial one(mainly, for loops), but is it possible that we can also apply such technique to recursion?

**Comment by cswiercz** [comment]
Yes! The lecture that I intended to give on Thursday was going to touch upon recursive algorithms. I will give said lecture this Tuesday, instead.
